course_info/templates/course_info/editor.html

```diff
@@ -86,47 +36,66 @@
 {% block content %}
 
     <div class="container-fluid">
-        <h4>Please select the fields you wish to display from the list below.</h4>
-
-        {# Display instructional text - some conditional on whether we're offering to insert text to the editor #}
-        <h5>
-            Then, select "Insert as Widget" to embed a live-updating widget to mirror your iSite
-            {% if should_offer_text %}
-                - or "Insert as Text" to place an editable table in the Canvas Editor
-            {% endif %}
-        </h5>
-
-        <p>This information is provided by the registrar's office.
-            If any of it is outdated or incorrect, please contact the catalog coordinator in your department.</p>
+      <p>
+        The course details you select below will be displayed on the page,
+        and will always reflect the most up-to-date information available
+        from the Registrar’s office.
+      </p>
     </div>
-    <br/>
-    <div id="widget_edit_form_container" class="container-fluid">
+
+    <div id="widget_edit_form_container" class="container-fluid course-info-edit-form">
 
         <form action="{{ launch_presentation_return_url }}" id="widget_edit_form" method="get">
 
+          <ul class="list-group">
             {% for field in fields %}
-                <div>
-                    <input type="checkbox" class="course_info_checkbox" id="{{ field.key }}" value="{{ field.key }}" checked/>
-                    <span>{{ field.value }}</span>
-                </div>
+                <li class="list-group-item checkbox">
+
+                  {% if field.key == 'course.registrar_code_display' %}
+                    {% if field.value and school_title %}
+                      <label>
+                        <input type="checkbox" class="course_info_checkbox" id="{{ field.key }}" value="{{ field.key }}" checked/>
+                        <strong>{{ school_title }}:</strong>
```

Comment:
To me it can be a little clearer when the entire `<label>` + `{{ field.value }}` structure and relationship are in a single logic block. Otherwise we have if/else wrapped around some closing tags only, and logic indentation can make seeing the structural tag relationships more difficult, like so:
```
<label>
  <input type="checkbox" class="course_info_checkbox" id="{{ field.key }}" value="{{ field.key }}" checked/>
  {% if field.value and school_title %}
      <strong>{{ school_title }}:</strong>
    </label>
    {{ field.value }}
  {% else %}
      <strong>{{ field.label }}:</strong>
      Field not populated by registrar
    </label>
  {% endif %}
```
I'll make the change to save a couple of lines of code, I guess. Not a big deal to me, hopefully it's still quite legible.